ReconstructOrder/datastructures/create_intensity_data.py

```diff
@@ -115,7 +115,7 @@ def chan_name_parser(imgs, img, chan_name):
                 imgs.replace_image(img, '640')
             elif any(substring in chan_name for substring in ['FM464', 'fm464']):
                 imgs.replace_image(img, 'ex561em700')
-        elif any(substring in chan_name for substring in ['BF', 'BrightField']):
+        elif any(substring in chan_name for substring in ['BF', 'BrightField', 'Default']):
```

Comment:
In Ivan and Janie's data, "Default" is used for the brightfield acquisition. I will temporarily leave this piece of code to be like this so that they are able to process the data.